riptide-opentracing/src/main/java/org/zalando/riptide/opentracing/OpenTracingPlugin.java

```diff
@@ -78,7 +78,11 @@
     private final SpanDecorator decorator;
 
     public OpenTracingPlugin(final Tracer tracer) {
-        this(tracer, LifecyclePolicy.composite(new ExplicitSpanLifecyclePolicy(), new NewSpanLifecyclePolicy()), new DefaultActivationPolicy(),
+        this(tracer,
+                LifecyclePolicy.composite(
+                        new ExplicitSpanLifecyclePolicy(),
+                        new NewSpanLifecyclePolicy()),
+                new DefaultActivationPolicy(),
```

Comment:
formatting is off here

Reply:
No, it's correct. `DefaultActivationPolicy` is a separate parameter, next to `LifecyclePolicy.composite` and `SpanDecorator.composite`.